src/bootstrap/native.rs

```diff
@@ -625,7 +625,15 @@ impl Step for TestHelpers {
         if builder.config.dry_run {
             return;
         }
-        let target = self.target;
+        // The x86_64-fortanix-unknown-sgx target doesn't have a working C
+        // toolchain. However, some x86_64 ELF objects can be linked
+        // without issues. Use this hack to compile the test helpers.
+        let target = if self.target == "x86_64-fortanix-unknown-sgx" {
+            //INTERNER.intern_string("x86_64-unknown-linux-gnu".into()).into()
```

Comment:
Please remove the stray commented code here.

Reply:
Fixed